CLIENT-SPECIFICATION.md

```diff
@@ -31,15 +33,17 @@ Pages are grouped by platform. A platform is an operating system. For example, `
 This section describes the standardised command-line interface (CLI) for clients implementing one. Clients that do not provide a CLI can ignore this section.
 
 ### Arguments
-A number of arguments MUST be supported (unless otherwise specified if a CLI is implemented:
+A number of arguments MUST be supported (unless otherwise specified) if a CLI is implemented:
 
-Argument		        | Meaning
-------------------------|--------------------
-`--update`				| Updates the offline cache of pages. MUST be implemented if cache is supported.
-`--version`, `-v`		| Shows the current version of the client, and the version of this specification that it implements.
-`--list`, `-l`			| Lists all the pages in the current platform to the standard output. If the special platform `all` is specified a list of all pages in all platforms MUST be displayed.
-`--pretty`				| OPTIONAL. Forces the output to contain all additional decorations, even if the standard output is not a TTY.
-`--platform`, `-p`		| Specifies the platform that should be used for resolving page names. If specified, the named platform MUST be checked first instead of the host platform  as described below.
+Argument		         | Meaning
+---------------------|--------------------
+`-v`, `--version`    | Shows the current version of the client, and the version of this specification that it implements.
+`-l`, `--list`       | Lists all the pages in the current platform to the standard output. If the special platform `all` is specified a list of all pages in all platforms MUST be displayed.
+`-p`, `--platform`   | Specifies the platform that should be used for resolving page names. If specified, the named platform MUST be checked first instead of the host platform as described below.
+`-t, `--translation` | Specifies the translation that should be used for resolving page names. If specified, the named language-tag MUST be checked first instead of the host language as described below.
```

Comment:
```suggestion
`-t`, `--translation` | Specifies the translation that should be used for resolving page names. If specified, the named language-tag MUST be checked first instead of the host language as described below.
```